src/viam/sdk/components/board.hpp

```diff
@@ -261,6 +267,7 @@ struct API::traits<Board> {
 };
 
 bool operator==(const Board::status& lhs, const Board::status& rhs);
+bool operator==(const Board::analog_value& lhs, const Board::analog_value& rhs);
```

Comment:
This line is only needed to get line 269 to work. Do we ever actually use the `Board::status` for anything? Part of me wonders if it was supposed to have been removed in https://github.com/viamrobotics/viam-cpp-sdk/pull/231, when `get_status` was removed. Maybe this is all dead code? Maybe not; maybe it needs to stay here.